incubator/validator-core/src/test/java/io/aklivity/zilla/runtime/validator/core/StringValidatorTest.java

```diff
@@ -14,121 +14,187 @@
  */
 package io.aklivity.zilla.runtime.validator.core;
 
-import static org.junit.Assert.assertFalse;
-import static org.junit.Assert.assertTrue;
+import static io.aklivity.zilla.runtime.engine.validator.FragmentValidator.FLAGS_COMPLETE;
+import static org.junit.Assert.assertEquals;
 
 import java.nio.charset.StandardCharsets;
 
 import org.agrona.DirectBuffer;
 import org.agrona.concurrent.UnsafeBuffer;
 import org.junit.Test;
 
+import io.aklivity.zilla.runtime.engine.validator.function.FragmentConsumer;
+import io.aklivity.zilla.runtime.engine.validator.function.ValueConsumer;
 import io.aklivity.zilla.runtime.validator.core.config.StringValidatorConfig;
 
 public class StringValidatorTest
 {
+    private static final int FLAGS_INIT = 0x02;
+
     @Test
     public void shouldVerifyValidUTF8()
```

Comment:
In class names and method names, use `InitCaps` for acronyms, so `shouldVerifyValidUtf8()`.